compyle/types.py

```diff
@@ -188,6 +188,7 @@ def _inject_types_in_module():
     NP_C_TYPE_MAP[np.dtype(np.uint64)] = 'unsigned long long'
     C_NP_TYPE_MAP['long long'] = np.int64
     C_NP_TYPE_MAP['unsigned long long'] = np.uint64
+    TYPES['long long'] = KnownType('long long')
```

Comment:
This fixes tests failing on windows but not sure if this is a good fix. All keys in the `TYPES` dict do not contain spaces as they are meant to be used with `@annotate` decorator?